src/components/Dao/DaoContainer.tsx

```diff
@@ -95,7 +95,7 @@ class DaoContainer extends React.Component<IProps, null> {
             <Route exact path="/dao/:daoAvatarAddress/proposal/:proposalId"
               render={(props) =>
                 <ProposalDetailsPage {...props}
-                  dao={dao}
+                  daoState={dao}
```

Comment:
this naming is confused throughout the app, should we always use state when referencing a IXXState type?